core/frontend/src/AccuDraw.ts

```diff
@@ -3147,4 +3196,64 @@ export class AccuDrawHintBuilder {
 
     return true;
   }
+
+  /** Create a [[Ray3d]] whose origin is the supplied space point and direction is into the view */
+  public static getBoresite(spacePt: Point3d, vp: Viewport, checkAccuDraw: boolean = false, checkACS: boolean = false): Ray3d {
+    return EditManipulator.HandleUtils.getBoresite(spacePt, vp, checkAccuDraw, checkACS);
+  }
+
+  /** Return ray intersection with a plane defined by a point and normal
+   * @see [[getBoresite]]
+   */
+  public static projectPointToPlaneInView(spacePt: Point3d, planePt: Point3d, planeNormal: Vector3d, vp: Viewport, checkAccuDraw: boolean = false, checkACS: boolean = false): Point3d | undefined {
+    return EditManipulator.HandleUtils.projectPointToPlaneInView(spacePt, planePt, planeNormal, vp, checkAccuDraw, checkACS);
+  }
+
+  /** Return ray intersection with a line defined by a point and direction
+   * @see [[getBoresite]]
+   */
+  public static projectPointToLineInView(spacePt: Point3d, linePt: Point3d, lineDirection: Vector3d, vp: Viewport, checkAccuDraw: boolean = false, checkACS: boolean = false): Point3d | undefined {
+    return EditManipulator.HandleUtils.projectPointToLineInView(spacePt, linePt, lineDirection, vp, checkAccuDraw, checkACS);
+  }
+
+  /** Return a [[Matrix3d]] representing the current working plane specified by AccuDraw, the view's ACS, or view rotation. */
+  public static getCurrentRotation(vp: Viewport, checkAccuDraw: boolean, checkACS: boolean, matrix?: Matrix3d): Matrix3d | undefined {
+    const current = AccuDraw.getCurrentOrientation(vp, checkAccuDraw, checkACS, matrix);
+    return (undefined !== current ? current.inverse() : undefined);
+  }
+
+  /** Return a [[Matrix3d]] corresponding to the supplied [[ContextRotationId]].
+   * A [[ContextRotationId]] that corresponds to a standard view, "Top", "Front", etc. will return a [[Matrix3d]] that
+   * is relative to the current view's ACS when ACS context lock is enabled.

```

Comment:
Here and elsewhere, most readers won't know what "ACS" means. Can you link to `AuxCoordSystemState`?

Reply:
done